### DISCUSSION TITLE
V2EX

### DISCUSSION THREAD

**Discussion by 1sm23** [elrumo/macOS_Big_Sur_icons_replacements#1183] [Help with the website]
Please help me with the PWA icon of this website https://www.v2ex.com
![](https://tva1.sinaimg.cn/large/008i3skNgy1gqzlbi44vwj3074074q2t.jpg)

**Comment by cake03** [comment]
<img alt="V2EX" src="https://user-images.githubusercontent.com/80169337/122051970-cd075b80-ce02-11eb-9653-0b8702377727.png" height="512" >
<!— comment —>

**Reply by 1sm23** [reply]
Much thanks !
